python/cufinufft/tests/test_basic.py

```diff
@@ -13,16 +13,16 @@ def _test_type1(dtype, shape=(16, 16, 16), M=4096, tol=1e-3):
 
     dim = len(shape)
 
-    kxyz = utils.gen_nu_pts(M, dim=dim).astype(dtype)
+    k = utils.gen_nu_pts(M, dim=dim).astype(dtype)
     c = utils.gen_nonuniform_data(M).astype(complex_dtype)
 
-    kxyz_gpu = gpuarray.to_gpu(kxyz)
+    k_gpu = gpuarray.to_gpu(k)
     c_gpu = gpuarray.to_gpu(c)
     fk_gpu = gpuarray.GPUArray(shape, dtype=complex_dtype)
 
     plan = cufinufft(1, shape, 1, tol, dtype=dtype)
 
-    plan.set_pts(M, kxyz_gpu[0], kxyz_gpu[1], kxyz_gpu[2])
+    plan.set_pts(M, k_gpu[2], k_gpu[1], k_gpu[0])
```

Comment:
Let's do `k_gpu[0], k_gpu[1], k_gpu[2]` here (same below). This works out if `make_grid` is also changed below.

Reply:
5a5aad9f